realtime-image-recognition/src/main/java/WebcamSource.java

```diff
@@ -80,7 +80,6 @@ public boolean isCooperative() {
         );
     }
 
-    @Override
```

Comment:
Inserting @cangencer's comment on email chain : Close method was recently added, so Override is correct.